applications/ColossalChat/coati/trainer/orpo.py

```diff
@@ -283,14 +283,22 @@ def _eval(self, epoch: int):
                 logprob_actor_reject = calc_masked_log_probs(
                     actor_reject_logits, reject_input_ids, reject_loss_mask[:, 1:]
                 )
-
-                odds_ratio_loss, log_odds_ratio = self.odds_ratio_loss_fn(logprob_actor_chosen, logprob_actor_reject)
-
+                chosen_logits = actor_chosen_logits[:, :-1, :].contiguous().view(-1, actor_chosen_logits.size(-1))
+                label_chosen = chosen_input_ids[:, 1:].contiguous()
```

Comment:
For labels, why do we need to shift manually? crossentropy loss should automatically shift and calculate loss?

Reply:
This is a bug, I will fix it, thanks